app/controllers/clients/enrollments_controller.rb

```diff
@@ -0,0 +1,28 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Clients
+  class EnrollmentsController < ApplicationController
```

Comment:
Which permissions should we required here @eanders

Reply:
Good question.
We should enforce that the enrollment is at a project you can see.
Let's also add `before_action :require_can_view_enrollment_details_tab!` since this is a different view of enrollment details.
Let's also only link to it if you have `can_view_enrollment_details_tab`

Reply:
Ok, done with https://github.com/greenriver/hmis-warehouse/pull/2468/commits/f344f3bff8e65961711406990208e66dd73e23e1.

> We should enforce that the enrollment is at a project you can see.
It already does this by using `visible_to`, it should 404 when it can't find the enrollment